src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting_NetFx/DocumentFormattingTest.cs

```diff
@@ -3870,17 +4669,17 @@ private void M()
                             };
                     
                             object gridOptions = new()
+                            {
+                                Columns = new GridColumn<WorkOrderModel>[]
                                 {
-                                    Columns = new GridColumn<WorkOrderModel>[]
-                                    {
-                                        new TextColumn<WorkOrderModel>(e => e.Name) { Label = "Work Order #" },
-                                        new TextColumn<WorkOrderModel>(e => e.PartNumber) { Label = "Part #" },
-                                        new TextColumn<WorkOrderModel>(e => e.Lot) { Label = "Lot #" },
-                                                new DateTimeColumn<WorkOrderModel>(e => e.TargetStartOn) { Label = "Target Start" },
-                                    },
-                                    Data = Model.WorkOrders,
-                                    Title = "Work Orders"
-                                };
+                                    new TextColumn<WorkOrderModel>(e => e.Name) { Label = "Work Order #" },
+                                    new TextColumn<WorkOrderModel>(e => e.PartNumber) { Label = "Part #" },
+                                    new TextColumn<WorkOrderModel>(e => e.Lot) { Label = "Lot #" },
+                                            new DateTimeColumn<WorkOrderModel>(e => e.TargetStartOn) { Label = "Target Start" },
+                                },
+                                Data = Model.WorkOrders,
+                                Title = "Work Orders"
+                            };
                         }
```

Comment:
These changes are a result of the new engine fixing https://github.com/dotnet/razor/issues/7933